drv/gimlet-seq-server/src/seq_spi.rs

```diff
@@ -21,6 +21,51 @@ pub enum Cmd {
 
 include!(env!("GIMLET_FPGA_REGS"));
 
+#[derive(AsBytes, Unaligned)]
+#[repr(u8)]
+#[allow(dead_code)]
+#[allow(non_camel_case_types)]
+pub enum A0StateMachine {
+    IDLE = 0x00,
+    PBTN = 0x01,
+    WAITSLP = 0x02,
+    GROUPB1_EN = 0x03,
+    GROUPB1_PG = 0x04,
+    GROUPB2_EN = 0x05,
+    GROUPB2_PG = 0x06,
+    GROUPC_PG = 0x07,
+    DELAY_1MS = 0x08,
+    ASSERT_PG = 0x09,
+    WAIT_PWROK = 0x0a,
+    WAIT_RESET_L = 0x0b,
+    DONE = 0x0c,
+    SAFE_DISABLE = 0x0d,
+}
+
+impl From<A0StateMachine> for u8 {
```

Comment:
Optional: if you add a dependency on `derive_more`, you could `derive(From)` and skip this implementation.

Reply:
hmmm, maybe not (see [this comment](https://jeltef.github.io/derive_more/derive_more/into.html#enums))

Reply:
Oh, love that! Thank you!